### ISSUE TITLE
[Feature Request/Question]: Exclude import suggestions of _laravel_idea directory

### ISSUE THREAD

**Issue by Allgoodnamesargone** [laravel-idea/plugin#619]
### Feature Description

Hello,
Well I am confused that something is wrong in my side or it's just like that!

I generate helper code by `Ctrl+Shift+.` and well, If i try to use for example `Arr:get(...)` on a new file i get a list of suggested classes to import which for me is contains `Arr` and `Illuminate\Support\Arr` which the second one is the correct import.
![image](https://user-images.githubusercontent.com/110684470/186148929-22ee702e-b28f-47a7-a307-457c00013374.png)
Is there any way to remove Import suggestions that are based in _laravel_idea directory?

**Comment by adelf** [comment]
They both are correct.

`\Arr::` is a standard alias for `\Illuminate\Support\Arr::`

**Comment by Allgoodnamesargone** [comment]
@adelf 
Correct, while you're typing `Arr` if you press `Enter` then that will use `\Arr`
But if you completely write `Arr::get('');` without pressing that `Enter`, it will just add a `use Arr;` and stays `Arr::get('');`

**Comment by adelf** [comment]
Now you can just opt out of the Aliases.

<img width="464" alt="Image" src="https://github.com/user-attachments/assets/66b3d18f-27eb-48d7-b5ec-3ddf0e8c4efb" />
